### ISSUE TITLE
Feature request: present websites using azul3d.org/semver.v1 nicely

### ISSUE THREAD

**Issue by emidoots** [shurcooL/Go-Package-Store#24]
**Background**

The semver package (see [GoDoc here](https://godoc.org/azul3d.org/semver.v1)) allows people to add support for semantic versioning of Go packages (basically what `gopkg.in` does) to custom domains (`foobar.org/foopkg.v3`).

Primarily it does mingling with the `go-import` meta tag and proxies git requests through the custom domain itself, so the meta tag doesn't contain a known code host (i.e. GitHub/Google Code) in it.

The meta tag for the `azul3d.org/audio.v1` package looks like this:

```
<meta name="go-import" content="azul3d.org/audio.v1 git https://azul3d.org/audio.v1">
```

Where the "git" repository at `https://azul3d.org/audio.v1` actually just proxies requests to `github.com/azul3d/audio`.

But the actual code host can be any arbitrary one (right now the package supports just GitHub, but it might support others like Google Code or private hosting in the future).

**TL;DR**

It would be cool if G-P-S could somehow talk with websites using `semver` to infer where they are hosted and present them as such. Basically:

| Import path | Hosted at | Presented by G-P-S as |
| --- | --- | --- |
| `azul3d.org/pkg.v1` | `github.com/azul3d/pkg` | a normal GitHub package |
| `azul3d.org/madness.v3` | `code.google.com/p/seperatehost` | a normal Google Code package |
| `foobar.org/zed.v34-dev` | `code.google.com/p/zeddev34` | a normal Google Code package |
| `evil.com` | private/non-public Git host | as G-P-S presents any other generic package |

There isn't anything in `semver` right now that particularly relates a site to any known code-hosting website.

I can add support to `semver` to show the underlying code host (i.e. the github repository) via e.g. a meta tag, but the question is what would be the best way to communicate this to G-P-S? I am open to any solution.
